omero/sysadmins/grid.txt

```diff
@@ -24,6 +24,9 @@ Requirements
 ^^^^^^^^^^^^
 
 The normal OMERO installation actually makes use of OMERO.grid internally.
+Note that multi-node OMERO.grid configurations are only tested with Unix-like
+systems, multi-node configurations on Windows are untested and deprecated,
+and any remaining support will be dropped in OMERO 5.3.
 If you have followed the instructions under :doc:`unix/server-installation`
 (Unix) or :doc:`windows/server-installation` (Windows), then you
```

Comment:
I'm find having the link to windows removed as @hflynn suggests, but if we need to be more radical in this paragraph (and potentially eliminate the "on Windows" section) we can